status.js

```diff
@@ -169,7 +169,7 @@ init
         jobset['update_time_relative'] = m.fromNow()
         jobset['update_time_local'] = m.format()
         // do not use color indications on outdated channels
-        if (jobset['current']) {
```

Comment:
as chatted in Matrix, let's calculate "current" once (like we do in the addition of this patch) and then reuse that value later.